tangelo/linq/tests/data/test_simulator.py

```diff
@@ -0,0 +1,505 @@
+# Copyright 2021 Good Chemistry Company.
```

Comment:
This file maybe should not exist. We already have `tangelo/linq/tests/test_simulator.py`.
This is currently in the `data` folder, which should not contain any code but only data files.